ext-apiserver/CONTRIBUTING.md

```diff
@@ -22,6 +22,10 @@ The machine controller should be able to act on a subset of machines that form a
 *   [Cluster type](https://github.com/kubernetes/kube-deploy/blob/master/ext-apiserver/pkg/apis/cluster/v1alpha1/cluster_types.go#L40)
 *   [Machine type](https://github.com/kubernetes/kube-deploy/blob/master/ext-apiserver/pkg/apis/cluster/v1alpha1/machine_types.go#L42)
 
+### Boostrapping
+
+To minimize code duplication and maximize flexibility, do boostrapping of clusters with an external Cluster Management API Stack (ie. all componenets needed to provide Kubernetes Cluster Management API for a cluster). [Bootsrap Process Design Details](https://docs.google.com/document/d/1CnzIXtitfbO6Y7ZxVWROGO8jr19t0vooDx-YQ7c2nbI/edit?usp=sharing).
```

Comment:
Please make the second sentence a proper sentence rather than a sentence fragment.